django/contrib/gis/db/backends/oracle/adapter.py

```diff
@@ -1,7 +1,43 @@
 from cx_Oracle import CLOB
 
 from django.contrib.gis.db.backends.base.adapter import WKTAdapter
+from django.contrib.gis.geos import Polygon, GeometryCollection
 
 
 class OracleSpatialAdapter(WKTAdapter):
     input_size = CLOB
+
+    def __init__(self, geom):
+        if isinstance(geom, Polygon):
+            self._fix_polygon(geom)
+        elif isinstance(geom, GeometryCollection):
+            self._fix_geometry_collection(geom)
+
+        self.wkt = geom.wkt
+        self.srid = geom.srid
+
+    def _fix_polygon(self, poly):
+        """Fix single polygon orientation for Oracle"""
```

Comment:
would be helpful to explain what the issue is (e.g. how must the polygon orientation be fixed)